src/screen-size-context-provider.tsx

```diff
@@ -27,8 +27,15 @@ const ScreenSizeContextProvider: React.FC<Props> = ({children}) => {
             tablet: stripMedia(theme.mq.tablet),
             tabletOrBigger: stripMedia(theme.mq.tabletOrBigger),
             tabletOrSmaller: stripMedia(theme.mq.tabletOrSmaller),
+            largueDesktop: stripMedia(theme.mq.largeDesktop),
         }),
-        [theme.mq.mobile, theme.mq.tablet, theme.mq.tabletOrBigger, theme.mq.tabletOrSmaller]
+        [
+            theme.mq.mobile,
+            theme.mq.tablet,
+            theme.mq.tabletOrBigger,
+            theme.mq.tabletOrSmaller,
+            theme.mq.largeDesktop,
```

Comment:
is this a new mediaquery? shouldn't it be a new definition part of this PR?

Reply:
No no, the mediaquery already exists, I'm just adding it to this hook to make it available to components, as the carousel requires it.